clusterDeployer.py

```diff
@@ -442,29 +434,15 @@ def create_workers(self) -> None:
         ssh_priv_key_path = self._get_discovery_ign_ssh_priv_key(infra_env)
         shutil.copyfile(ssh_priv_key_path, os.path.join(_BF_ISO_PATH, "ssh_priv_key"))
 
-        futures = []
-        for h in hosts_with_workers:
-            futures.extend(h.start_workers(iso_file, infra_env, executor))
-
-        # Wait for workers to have booted.
         for h in hosts_with_workers:
-            h.wait_for_workers_boot(self._cc.full_ip_range)
+            h.ensure_images(iso_file, infra_env, masters=False)
```

Comment:
Hmm, before this change the images were copied in parallel (as part of start_workers). We now do it one host at a time unless I'm missing something.

Should we wrap this into futures that execute in parallel too?

But we can do it in a follow up PR too.

Reply:
Good point, I'll try to address it within this PR.